cricPredict/User_profile/TokenValidationMiddleware.py

```diff
@@ -0,0 +1,22 @@
+from django.core.cache import cache
+from django.core.exceptions import ValidationError
+
+
+class BaseMiddleware(object):
+    def __init__(self, get_response):
```

Comment:
why do we need this empty class? can you instead explore `MiddlewareMixin`?

Reply:
Okay